test/ooni/tcp_test.cpp

```diff
@@ -48,10 +48,10 @@ TEST_CASE("TCPTestImpl works as expected in a common case") {
                           {"port", "80"},
                           {"dns/nameserver", "8.8.8.1"},
                           {"dns/timeout", 0.0001}},
-                         [&count](Error err, Var<net::Transport> txp) {
+                         [&count](Error err, Var<net::Transport>) {
                              REQUIRE(err);
                              if (++count >= 3) {
-                                 []() { break_loop(); };
+                                 break_loop();
```

Comment:
Because the [other test was ablet to call break loop](https://github.com/measurement-kit/measurement-kit/pull/608/files#diff-01d046af7ec7bc1862230cb8263d88e5R41)